cmd/loadtest/main.go

```diff
@@ -6,35 +6,71 @@ package main
 import (
 	"os"
 	"strings"
+	"time"
 
 	"github.com/mattermost/mattermost-load-test-ng/config"
 	"github.com/mattermost/mattermost-load-test-ng/example"
 	"github.com/mattermost/mattermost-load-test-ng/loadtest"
+	"github.com/mattermost/mattermost-load-test-ng/loadtest/control"
+	"github.com/mattermost/mattermost-load-test-ng/loadtest/control/simplecontroller"
+	"github.com/mattermost/mattermost-load-test-ng/loadtest/store/memstore"
+	"github.com/mattermost/mattermost-load-test-ng/loadtest/user/userentity"
+
 	"github.com/mattermost/mattermost-server/v5/mlog"
 	"github.com/spf13/cobra"
 )
 
 func RunLoadTestCmdF(cmd *cobra.Command, args []string) error {
-	return loadtest.Run()
+	config, err := config.GetConfig()
+	if err != nil {
+		return err
+	}
+
+	newSimpleController := func(id int, status chan<- control.UserStatus) control.UserController {
+		ueConfig := userentity.Config{
+			ServerURL:    config.ConnectionConfiguration.ServerURL,
+			WebSocketURL: config.ConnectionConfiguration.WebSocketURL,
+		}
```

Comment:
I wonder, since loadtest is ran against one instance at a time - what's the point of storing ServerURL and WebsocketURL in each userEntity? Why not keep in some global/context?

Reply:
In a sense we already have this global object which is the config.  
The idea of passing `ServerURL`  and  `WebSocketURL` there was to keep the lower level decoupled from the config and every user's state completely independent from each others. If it ever becomes a problem of memory we have several possible solutions but for now I am not too worried about this kind of redundancy.